src/types/ErrorMessages.ts

```diff
@@ -0,0 +1,10 @@
+export enum ErrorMessages {
+  NONE = '',
+  LOAD = 'Unable to load todos',
+  TITLE = 'Title can\'t be empty',
+  ADD = 'Unable to add the todo',
+  DELETE = 'Unable to delete the todo',
+  DELETE_COMPLETED = 'Unable to delete completed todos',
+  UPDATE = 'Unable to update a todo',
+  UPDATE_ALL = 'Unable to update status of todos',
```

Comment:
I am not sure that it is a good idea to store whole sentences in enum, maybe it's better to check for ErrorMessages value in your ErrorMessage.tsx and show these sentences there.